### DISCUSSION TITLE
Install ansible-lint with yamlint support, point to older version.

### DISCUSSION THREAD

**Discussion by jer117** [ansible/ansible-lint#1639] [Q&A]
Hi, 

I want ot install an old version of ansible-lint with yamllint support. 
> ANSIBLE_LINT_VERSION=4.2.0
> pip3 install "ansible-lint==${ANSIBLE_LINT_VERSION}[yamllint]"
I want everything to function the same was as the following command. 
> pip3 ininstall "ansible-lint[yamllint]"
I have tried doing this but it's not the same. 
> pip3 install "ansible-lint==${ANSIBLE_LINT_VERSION}" && pip3 install yamllint. 
Any workaround would be great.

**Comment by konstruktoid** [comment]
```sh
$ pip3 install "ansible-lint[yamllint]"==4.2.0
Collecting ansible-lint[yamllint]==4.2.0
  Using cached ansible_lint-4.2.0-py2.py3-none-any.whl (57 kB)
WARNING: ansible-lint 4.2.0 does not provide the extra 'yamllint'
Collecting ansible>=2.7
[...]
```

**Reply by konstruktoid** [reply]
Since `4.2.0 does not provide the extra 'yamllint'`  I assume `pip3 install "ansible-lint==${ANSIBLE_LINT_VERSION}" && pip3 install yamllint` is the way to go.
